src/errors.rs

```diff
@@ -86,7 +86,10 @@ pub enum RustupError {
     RunningCommand { name: OsString },
     #[error("toolchain '{0}' is not installable")]
     ToolchainNotInstallable(String),
-    #[error("toolchain '{0}' is not installed")]
+    #[error("toolchain '{0}' is not installed.{}", match .0.to_string().parse::<PartialToolchainDesc>() {
+        Ok(desc) => format!("\nhelp: run `rustup toolchain install {}` to install it.", .0),
```

Comment:
I don't think you should be adding the `.` at the end of each line, that's not coherent with the rest of the codebase.